tests/e2e/specs/factory/RefusedOAuthFactory.spec.ts

```diff
@@ -185,20 +200,55 @@ suite(
 				await scmContextMenu.select('Changes', 'Stage All Changes');
 			});
 
-			test('Commit the changes', async function (): Promise<void> {
+			test('Get modal dialog when try to commit changes', async function (): Promise<void> {
 				Logger.info(`ScmView inputField locator: "${(webCheCodeLocators.ScmView as any).scmEditor}"`);
 				Logger.debug('Click on the Scm Editor');
 				await driverHelper
 					.getDriver()
 					.findElement((webCheCodeLocators.ScmView as any).scmEditor)
 					.click();
+				Logger.debug(`Type commit text: "Commit ${changesToCommit}"`);
```

Comment:
This code is a good candidate to extract this code to the specific method which handles commit from VS Code Editor.